src/vehicle.cpp

```diff
@@ -1505,7 +1505,7 @@ int vehicle::install_part( const point &dp, const vehicle_part &new_part )
         }
     }
     // refresh will add them back if needed
-    remove_fake_parts( false );
+    remove_fake_parts( true );
```

Comment:
I'm only passing on this right now because adjusting these remove parts calls when debugging the game parts feature caused no end of errors and i don't want to risk breaking things worse right before a release.

Reply:
@kevingranade are we good to merge this now then and see if there are cascading effects?

Reply:
Yep,  just haven't gotten around to it yet.